lib/Doctrine/Migrations/Metadata/ExecutedMigration.php

```diff
@@ -26,11 +26,22 @@ final class ExecutedMigration
      */
     public $executionTime;
 
-    public function __construct(Version $version, ?DateTimeImmutable $executedAt = null, ?float $executionTime = null)
-    {
+    /** @var string|null */
+    private $reason;
+
+    /** @var string|null */
+    private $reasonDescription;
+
+    public function __construct(
+        Version $version,
+        ?DateTimeImmutable $executedAt = null,
+        ?float $executionTime = null,
+        ?string $reason
```

Comment:
Why did you decide not to use
```suggestion
        ?string $reason = null
```
?